Libraries/Opc.Ua.Client/Session.cs

```diff
@@ -1482,7 +1482,7 @@ public void FetchNamespaceTables()
             ResponseHeader responseHeader = this.Read(
                 null,
                 0,
-                TimestampsToReturn.Both,
```

Comment:
Is this change intended?
Seems to have nothing to do with IClone.

Reply:
Thats correct. Intended yes, but not for IClone. Thanks for checking...

Reply:
Its a tiny optimizitation, because the timestamps are not needed here.

Reply:
Ok, thanks.